examples/easylisten.rs

```diff
@@ -1,6 +1,14 @@
 fn main() {
     // get the default Device
-    let mut cap = pcap::Device::lookup().unwrap().open().unwrap();
+    let device = pcap::Device::lookup().unwrap();
+    println!("Using device {}", device.name);
```

Comment:
I like this!

Reply:
Yea, I found it harder to debug without this print statement!